pyproject.toml

```diff
@@ -1,6 +1,13 @@
 [tool.ruff]
 line-length = 120
 
+select = [
+    "E",  # Pycodestyle
+    "F",  # Pyflakes
+    "PL", # pylint
+]
+ignore = ["PLR0913"]
```

Comment:
If this is rule is broken in 1-2 places, should we add this inline rather than for the whole project here?

Reply:
We explicitly disabled this in pylint: https://github.com/aws/aws-lambda-builders/blob/02b03aa91bc19a83f215e48c0b8a99c7a154afe5/.pylintrc#L64

We can but not sure if that there is a ton of value here. This is for the max args into a function.